ws-nextjs-app/pages/[service]/live/[id]/Stream/index.tsx

```diff
@@ -21,6 +21,22 @@ const Stream = ({
     },
   } = useContext(ServiceContext);
 
+  const [hasShareApi, setHasShareApi] = useState(false);
+  const [hashValue, setHashValue] = useState('');
+
+  useEffect(() => {
+    const URLHash = window.location.hash.substring(1);
+    setHashValue(URLHash);
+
+    if (hashValue) {
+      window.location.href = `#${hashValue}`;
+    }
+    setHasShareApi(true);
```

Comment:
Should this be here?

Reply:
Ahh sorry this is just for testing temporarily, will remove before go live

Reply:
This will also cause a unit test to fail for now